lib/galaxy/webapps/tool_shed/app.py

```diff
@@ -10,9 +10,11 @@
 from galaxy.openid.providers import OpenIDProviders
 from galaxy.util.dbkeys import GenomeBuilds
 from galaxy.web import security
+from galaxy.config import configure_logging
```

Comment:
There are 3 principles:
- 3 sections, separated by an empty line:
  1) standard library imports;
  2) external libraries imports;
  3) internal and relative imports.
- Inside each sections, all `import lib` go before the `from lib import foo`
- Then sort alphabetically each subsection

Example: https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_smarkets.py